poscreators/integration-checklist.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: /poscreators/get-started/checklist
+title: Integration Checklist
+---
+
+# Integration Checklist
+Integrating the Middleware involves several steps, from calling the API the right way to registering in the production Portal and connecting to Dealers. To simplify this process for POS Creators, we've composed a concise checklist that can be used to validate if all required steps were succesfully performed.
+
+| Country | Check | Description | Docs |
+| :----: | ----- | ----------- | --------- |
+| ![All markets](images/flag-eu.png) | **Sandbox Portal account created** | An account in the sandbox Portal was created to create test Middleware instances. | [Getting Started Guide](README.md) |
+| ![All markets](images/flag-eu.png) | **Production Portal account created & partner contract signed** | An account in the production Portal was created to manage POS Systems, and the PosCreator partner contract was signed. | [Getting Started Guide](README.md) |
```

Comment:
In France we use always the term "and PosCreator role is activated" instead of "contract is signed".
We have got experience with this and it leads more to a partner-relationship than customer-relationship. It's the first step to the long road of *Customer Success*